src/processors/BaseProcessor.h

```diff
@@ -161,5 +172,9 @@ class BaseProcessor : private JuceProcWrapper
     StringArray popupMenuParameterIDs;
     OwnedArray<ParameterAttachment> popupMenuParameterAttachments;
 
+    bool modulationPorts = false;
```

Comment:
I'm not sure we need this member variable.

Reply:
Right, thanks for catching that